### ISSUE TITLE
Mental wellbeing feature: showing new notifications in bulk at a specific time of day

### ISSUE THREAD

**Issue by sergia-ch** [mastodon/mastodon#23015]
### Pitch

It would be nice to be able to make notifications less addictive. One of the reason they are is the short-term effect (shown immediately as another person favourites or reblogs), and refreshing them manually expecting an uncertain reward. This setting is known to create an obsessive response (I took neuroscience classes and we were shown mice going crazy in such a scenario I think)

One of the solutions is to queue new notifications. Say, show new ones once per day or once per 3 hours say exactly at 1, 4 and 7pm. This way the reward would be less uncertain, as it can only appear at a pre-defined time.

The app setting which changes notification periodicity also takes effect with a delay, so it cannot be changed during a little obsessive episode when one wants to see if their posts are liked.

I feel it has to be done at the backend level, otherwise it would be easy to just change the settings in the frontend during that obsessive "let's just look what's there" episode.

I'm willing to implement it myself. I'm asking if it's something that is likely to get merged, or if it's something that Mastodon is not about.

I have skills, I co-founded an ethical recmmendations nonprofit, interned at Google and have an MSc degree from a Swiss uni.

### Motivation

I'm a mentally vulnerable person, with ADHD, OCD, depression, anxiety and histrionic personality disorder. I use Mastodon for discussing tech politics, activism, and sometimes looking for jobs.

Despite all my efforts to make this experience not obsessive, it's hard for me to do so. My mother left me when I was 6 and my reward mechanisms in my brain are all f-ed up. My whole life is like this, I'm sometimes obsessing about buying some groceries for hours, and sometimes remembering a conversation I had 10 years ago and writing to a person about it. I am in therapy for years.

For me this could be something good, and a deal breaker.

I suspect that for a lot of people with mental health issues this could be important too. It could be the first step for Mastodon in that direction.

In truth, I don't really know. I want to do it together with interested people, not alone pushing my lonely idea. Looking forward to feedback, good and bad, and a discussion. Thank you.

**Comment by RonaldZielaznicki** [comment]
Yes to everything about ADHD mental health. I've the same problems, though untested since I'm new to Mastodon.

I'm here to add an additional incentive for this feature / suggestion.

Allowing server owners to make notifications queue into one message would reduce the number of emails sent out. Mailgun, sendgrid, and similar services have limits on the number of emails sent out. Queuing emails would save server owners money.
